products/jira.md

```diff
@@ -170,6 +314,405 @@ releases:
+    latestReleaseDate: 2018-06-26
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.10.x+release+notes
+
+-   releaseCycle: "7.9"
+    releaseDate: 2018-04-08
+    eol: 2020-04-09
+    latest: "7.9.2"
+    latestReleaseDate: 2018-05-08
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.9.x+release+notes
+
+-   releaseCycle: "7.8"
+    releaseDate: 2018-02-19
+    eol: 2020-02-20
+    latest: "7.8.4"
+    latestReleaseDate: 2018-05-08
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.8.x+release+notes
+
+-   releaseCycle: "7.7"
+    releaseDate: 2018-01-10
+    eol: 2020-01-11
+    latest: "7.7.4"
+    latestReleaseDate: 2018-05-09
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.7.x+release+notes
+
+-   releaseCycle: "7.6"
+    releaseDate: 2017-11-15
+    lts: true
+    eol: 2019-11-14
+    latest: "7.6.17"
+    latestReleaseDate: 2019-11-03
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.6.x+release+notes
+
+-   releaseCycle: "7.5"
+    releaseDate: 2017-09-05
+    eol: 2019-09-06
+    latest: "7.5.4"
+    latestReleaseDate: 2018-01-16
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.5.x+release+notes
+
+-   releaseCycle: "7.4"
+    releaseDate: 2017-06-28
+    eol: 2019-06-29
+    latest: "7.4.6"
+    latestReleaseDate: 2018-01-15
+    link:
+      http://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.4.x+release+notes
+
+-   releaseCycle: "7.3"
+    releaseDate: 2017-01-02
+    eol: 2019-01-03
+    latest: "7.3.9"
+    latestReleaseDate: 2018-01-10
+    link:
+      https://confluence.atlassian.com/jirasoftware/issues-resolved-in-7-3-9-943532124.html
+
+-   releaseCycle: "7.2"
+    releaseDate: 2016-08-23
+    eol: 2018-08-23
+    latest: "7.2.15"
+    latestReleaseDate: 2018-06-05
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.2.x+release+notes
+
+-   releaseCycle: "7.1"
+    releaseDate: 2016-02-10
+    eol: 2018-02-10
+    latest: "7.1.10"
+    latestReleaseDate: 2016-09-14
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.1.x+release+notes#JIRASoftware7.1.xreleasenotes-7-1-10
+
+-   releaseCycle: "7.0"
+    releaseDate: 2015-10-01
+    eol: 2017-10-06
+    latest: "7.0.11"
+    latestReleaseDate: 2016-09-13
+    link:
+      https://confluence.atlassian.com/display/JIRASOFTWARE/JIRA+Software+7.0.x+release+notes#JIRASoftware7.0.xreleasenotes-7-0-11
+
+-   releaseCycle: "6.4"
+    releaseDate: 2015-04-07
+    eol: 2017-03-17
+    latest: "6.4.14"
+    latestReleaseDate: 2016-07-28
+    link:
+      https://confluence.atlassian.com/jira064/jira-6-4-14-release-notes-834232021.html
+
+-   releaseCycle: "6.3"
+    releaseDate: 2014-07-09
+    eol: 2016-07-08
+    latest: "6.3.15"
+    latestReleaseDate: 2014-08-11
```

Comment:
```suggestion
    latestReleaseDate: 2015-02-03
```

According to https://confluence.atlassian.com/jira063/jira-6-3-15-release-notes-707625090.html.